client/snap_op.go

```diff
@@ -94,7 +94,12 @@ func (client *Client) Revert(name string, options *SnapOptions) (changeID string
 	return client.doSnapAction("revert", name, options)
 }
 
+var ErrNoDangerous = fmt.Errorf("dangerous option only valid when installing from a file")
```

Comment:
Unrelated to this branch but I think at some point we need to think about i18n for those errors.